src/vs/vscode.d.ts

```diff
@@ -3738,6 +3738,10 @@ declare module 'vscode' {
 		 * This rule will only execute if the text after the cursor matches this regular expression.
 		 */
 		afterText?: RegExp;
+		/**
+		 * This rule will only execute if the text above the this line matches this regular expression.
+		 */
+		oneLineAboveText?: RegExp;
```

Comment:
fyi @jrieken this PR adds a new property to `OnEnterRule` in `vscode.d.ts`.

Reply:
@alexandrudima please create a tracking item so we know this is an API change